### PR TITLE
Fix bug 1211010 - Private Browsing home page promo

### PR DESCRIPTION
**Pull request by craigcook**
Still needs a graphic.

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 1]
```diff
@@ -228,7 +228,17 @@ <h2 class="primary go">{{ _('Teach the Web') }}</h2>
           </a>
         </li>
       {% endif %}
-      {% if l10n_has_tag('fx10_independent') %}
+      {% if l10n_has_tag('firefox-private-browsing') %}
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/home/home.html]: Please use underscores for tags ```firefox_private_browsing```

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 2]
```diff
@@ -228,7 +228,17 @@ <h2 class="primary go">{{ _('Teach the Web') }}</h2>
...
+        <li id="promo-10" class="item promo-large-portrait private-browsing" tabindex="0">
+          <h2 class="primary go">{{ _('Introducing the new Private Browsing') }}</h2>
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/home/home.html]: Just to confirm: no line breaks here?

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 3]
```diff
@@ -228,7 +228,17 @@ <h2 class="primary go">{{ _('Teach the Web') }}</h2>
...
+          <a class="panel-link" href="{{ url('firefox.developer') }}" data-promotion-id="promo-10" data-promotion-name="Introducing the new Private Browsing" data-promotion-class="promo-large-portrait" data-promotion-type="tile">
+            <div class="secondary">
+              <h3>{{ _('More protection.') }}<br>{{ _('The most privacy.') }}</h3>
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/home/home.html]: Please use one string: ```{{ _('More protection. <br>The most privacy.') }}```

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 4]
```diff
@@ -228,7 +228,17 @@ <h2 class="primary go">{{ _('Teach the Web') }}</h2>
...
+              <p class="more">{{ _('Try private browsing') }}</p>
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/home/home.html]: This should be `Try Private Browsing`, both based on copy deck and the landing page.

**Comment by flodolo** [comment]
Temporary strings extracted, waiting for next push to double check.

**Comment by flodolo** [comment]
Final strings extracted, thanks unsleeping Craig.

**Comment by craigcook** [comment]
:sleeping:

**Comment by craigcook** [comment]
Added graphic, this should be mergeable now. It uses the `tracking-protection` waffle switch so it'll go live with everything else behind that switch.

**Comment by craigcook** [comment]
Spoke too soon, this is blocked by #3391, which needs to merge first.

**Comment by alexgibson** [comment]
Arrr + :boat:

**Pull request merged**